saga/sagalogs/file_saga_log_test.go

```diff
@@ -0,0 +1,162 @@
+package sagalogs
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"reflect"
+	"testing"
+
+	"github.com/scootdev/scoot/saga"
+)
+
+func getDirName() string {
+	path, err := os.Getwd()
```

Comment:
github.com/scootdev/scoot/os/temp lets this go in a temp dir, which is nice to avoid spurious writes in our dev tree.